src/control/cmd/dmg/storage_query.go

```diff
@@ -150,3 +151,46 @@ func (cmd *listPoolsQueryCmd) Execute(_ []string) error {
 	}
 	return cmd.makeRequest(ctx, req, control.PrintWithVerboseOutput(cmd.Verbose))
 }
+
+// usageQueryCmd is the struct representing the scan storage subcommand.
+type usageQueryCmd struct {
+	logCmd
+	ctlInvokerCmd
+	hostListCmd
+	jsonOutputCmd
+	//Verbose    bool `short:"v" long:"verbose" description:"List SCM & NVMe device details"`
```

Comment:
Is verbose coming in this PR or another? Or just part of an earlier draft?

Reply:
have removed verbose